students/mariusz_michalczyk/lesson_05_lists/comma_code.py

```diff
@@ -0,0 +1,9 @@
+spam = ['apples', 'bananas', 'tofu', 'cats']
+
+
+def get_comma_string(my_list):
+    my_list[-1] = "and " + my_list[-1]
```

Comment:
I was wondering if this shouldn't require the list to be at least 2 elements long, but they didn't specify it, so let it be.